site/content/en/guides/producer/substitutions/_index.md

```diff
@@ -89,7 +89,7 @@ spec:
 
 ```sh
 # create an image substitution and a setter that populates it
-kpt cfg create-subst hello-world/ image-tag nginx:1.7.9 \
+kpt cfg create-subst hello-world/ image-value nginx:1.7.9 \
```

Comment:
also update the other places `image-tag` appears as the substitution name

Reply:
This is the only place that `image-value` is used in this file.

Reply:
Search for the string `image-value`:

line 105, 108, 125

Reply:
Woops, you are correct